testsuite/httpx/__init__.py

```diff
@@ -72,9 +73,12 @@ def __init__(self, *, verify: Union[Certificate, bool] = True, cert: Certificate
         self.retry_codes = {503}
         _verify = None
         if isinstance(verify, Certificate):
-            verify_file = create_tmp_file(verify.chain)
-            self.files.append(verify_file)
-            _verify = verify_file.name
+            if verify.chain is None:
```

Comment:
I don't like this idea since it always defaults to letsencrypt cert for all certificates with undefined chain, which is wrong.